shell/platform/fuchsia/flutter/pointer_injector_delegate.h

```diff
@@ -129,6 +133,8 @@ class PointerInjectorDelegate {
 
     void EnqueueEvent(fuchsia::ui::pointerinjector::Event event);
 
+    void Reset();
```

Comment:
quick explanation pls - maybe mention what specific pieces of state get reset, and why those.

https://google.github.io/styleguide/cppguide.html#Function_Comments

Reply:
Done